WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -167,6 +168,8 @@ - (void)viewWillAppear:(BOOL)animated
     [self.navigationController setNavigationBarHidden:NO animated:NO];
     [self.navigationController setToolbarHidden:YES];
     
+    self.navigationController.view.tintColor = [UIColor murielEditorPrimary];
```

Comment:
What does this mutation address? Child views?

Reply:
Marking this as resolved, as I found the explanation you provided in the PR description. Thank you for that. 🙇🏻 

> Post Settings Screen
>
> The tintColor was set for multiple UI elements in [PostSettingsViewController](https://github.com/wordpress-mobile/WordPress-iOS/blob/4763d9c0ca69c2df63aa5e2b2b82e4928c8a23ba/WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m), including its [main view](https://github.com/wordpress-mobile/WordPress-iOS/blob/4763d9c0ca69c2df63aa5e2b2b82e4928c8a23ba/WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m#L144), the [subviews it's possible to navigate to](https://github.com/wordpress-mobile/WordPress-iOS/blob/4763d9c0ca69c2df63aa5e2b2b82e4928c8a23ba/WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m#L171), and the [sticky toggle](https://github.com/wordpress-mobile/WordPress-iOS/blob/4763d9c0ca69c2df63aa5e2b2b82e4928c8a23ba/WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m#L754).